### PR TITLE
Added test class to Learn command

### PR DESCRIPTION
**Pull request by lechemon4000**

### CODE REVIEW

File: src/main/java/org/fundacionjala/at15/pokemon/commands/behavior/Learn.java [hunk 1]
```diff
@@ -35,11 +36,13 @@ public Integer call() {
             Writer.writeToJson(pokemon);
 
             System.out.println("Your Pokemon learn a new Attack movement");
-        } else {
-            System.out.println("Arguments are invalid");
         }
+        // } else {
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/pokemon/commands/behavior/Learn.java:40]: avoid comments

Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/pokemon/commands/behavior/Learn.java:40]: you should remove these comments

File: src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandLearnTest.java [hunk 2]
```diff
@@ -1,24 +1,32 @@
 package org.fundacionjala.at15.pokemon.commands;
 import static org.junit.Assert.assertEquals;
```
Comment (adhes14) [inline src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandLearnTest.java:3]: You must import:
`import org.fundacionjala.at15.pokemon.commands.create.CreateCurrentEntities;`

File: src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandLearnTest.java [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
-    //@Test
+    @Test
     public void commandShouldCreateAddMovementToJsonFile() {
-        CreatePokemon pokemon1 = new CreatePokemon();
-        new CommandLine(pokemon1).parseArgs("-n", "pikachu", "-hp", "200");
-        pokemon1.call();
+        CreatePokemon poke = new CreatePokemon();
+        new CommandLine(poke).parseArgs("-n", "pikachu", "-hp", "100");
+        poke.call();
+
```
Comment (adhes14) [inline src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandLearnTest.java:19]: Before using a select command you must create a currents json files (crt-12345678):
`CreateCurrentEntities current = new CreateCurrentEntities();`
`new CommandLine(current);`
`current.call();`

**Review by adhes14 — CHANGES_REQUESTED** [review]
I suggested you some changes in order to fix your problem

File: src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandLearnTest.java (hunk 3 above)
Comment (lechemon4000) [reply src/test/java/org/fundacionjala/at15/pokemon/commands/SubCommandLearnTest.java:21]: Thanks Adhemar! now is working the test

**Review by adhes14 — DISMISSED** [review]
Well done!

**Review by vladitm19 — APPROVED** [review]

**Comment by Dru27** [comment]
Good job!

**Pull request merged**